src/Umbraco.Examine/ExamineExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static class ExamineExtensions
         /// <remarks>
         /// myFieldName_en-us will match the "en-us"
         /// </remarks>
-        internal static readonly Regex CultureIsoCodeFieldNameMatchExpression = new Regex("^([_\\w]+)_([a-z]{2}-[a-z0-9]{2,4})$", RegexOptions.Compiled);
+        internal static readonly Regex CultureIsoCodeFieldNameMatchExpression = new Regex("^(?<FieldName>[_\\w]+)_(?<CultureName>[a-z]{2,3}(-[a-z0-9]{2,4})?)$", RegexOptions.Compiled | RegexOptions.ExplicitCapture);
```

Comment:
The main fix is making `-[a-z0-9]{2,4}` optional, but as that also increased the group count from 3 to 4 (and this was checked by some implementations).

To make this regex easier to use and even backwards-compatible, I've named the parts for the field name and culture name and added `RegexOptions.ExplicitCapture`. This results is the same group count, order/index and even allows getting the right values by group name 🎉

Reply:
I've also changed the neutral culture name to be 2 **or 3** characters long.

Some specific cultures might even include a third part (e.g. `az-Cyrl-AZ` or `az-Latn-AZ`), but those aren't matched yet. These cultures are probably not used very often, so to avoid making the regex more complex, slower and possibly match even more non-culture fields, I've omitted that for now.